helpers/filterSeats.js

```diff
@@ -15,11 +14,11 @@ module.exports = function( dust ) {
   * @example {@_filterSeats packageRates=packageRatesReply.packageRates venueProducts=venueProductsReply.venueProducts roomRates=packageRatesReply.linked.roomRates ticketRates=packageRatesReply.linked.ticketRates} {/_filterSeats} output loop of sections with seats inside sorted by price
   */
 
-  dust.helpers._filterSeats = function( chunk, context, bodies, params ) {
+  dust.helpers._filterSeats = function(chunk, context, bodies, params) {
     params = params || {};
 
     var packageRates = params.packageRates;
-    var venueProduct = _.first( params.venueProducts );
+    var venueProduct = _.head(params.venueProducts);
```

Comment:
If you think it's not worth adding this let me know, I'm not familiar with this repo and will go with your judgement, especially if we're going to archive this one...